### DISCUSSION TITLE
Recommended way to find specific fetcher in useFetchers when submission is null

### DISCUSSION THREAD

**Discussion by ConnorVO** [remix-run/remix#2134] [Q&A]
I am making an API call from Component1 to get some data. I want to use that data in Component2 (sibling component). I am getting this data with useFetcher and should be able to share it with other components using useFetchers. The example in the docs is for a submission and they use the submission's action path to identify the proper fetcher. If you aren't using a submission in the fetcher, how should you id the proper fetcher? Attach some data to the fetcher to id it (i.e. add {'id': 'id-to-find-this-fetcher'}?

As I type this out, I guess I should just get that data in the parent component and pass it to each child, but I think my question still stands for other use cases

**Comment by Mxs2019** [comment]
Facing the exact same issue. I have a Form abstraction and would love to be able to add an ID to a fetcher so that I can easily find the response via useFetcher
